mantidproject/mantid
# Add Eigen as an external project

Add Eigen as an external dependency.

**To test:**

Checkout the branch and re-run `cmake` however you normally would. Check that there are no errors.

Try running `cmake` in a clean directory and check there are no errors.

You should also checkout out commit 9de61cf and try building the `Beamline` target. You can see the build history to see that this test passed on the build servers (`Eigen` will soon be used in `Beamline`).

Check no extra targets are created, e.g. `ninja -t targets` or check the list in VS.

Fixes #18540.

*Internal change, does not need to be in the release notes - but developer documentation should be updated once this is merged.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
